components/html-block/html-block.js

```diff
@@ -76,16 +76,22 @@ export const htmlBlockContentStyles = css`
 	ul, ul[type="disc"] {
 		list-style-type: disc;
 	}
-	ul ul, ul ol,
```

Comment:
Not a big deal, but I would liked the flexibility of allowing this, in the effort to reduce unnecessary scrolling in some cases. 🤷‍♂️

Reply:
Haha I'm on the flip side... I find if rules get combined onto 1 line like this I can easily miss stuff.

Reply:
I'm actually shocked there weren't more cases of this, so I think that's encouraging that enforcing one per line is probably good

Reply:
Yep... I can appreciate that. 😄

Reply:
Hmmm, is it only a rule that applies to selectors using combinators? I notice the a couple immediately above that are unchanged.

Reply:
It applies _if_ it's already a multi-line selector list. So,
```
// pass
a, b, c
```
```
// pass
a,
b,
c
```
```
// fail
a, b,
c
```

Reply:
Ah interesting, that'd make more sense, I really thought we did this all over the place.  Like [this](https://github.com/BrightspaceUI/core/blob/3fb26c72b6a845d6689b681a2ffb40e1862887f3/components/html-block/html-block.js#L26).

Reply:
Ok that sounds like the best of both worlds then!